app/views/messages/create.turbo_stream.erb

```diff
@@ -0,0 +1,21 @@
+<%# Add thinking indicator to messages area %>
+<%= turbo_stream.append "messages" do %>
+  <%= render "chats/thinking_indicator", chat: @chat %>
+<% end %>
+
+<%# Reset the form by replacing it with a fresh version %>
+<%= turbo_stream.replace "chat-form" do %>
+  <%= render "messages/chat_form", chat: @chat %>
+<% end %>
+
+<%# Focus the input after form replacement %>
+<%= turbo_stream.after "chat-form" do %>
```

Comment:
Generally prefer to not use `<script>` tags in this way. Should make use of Rails javascript conventions with Stimulus.